### PR TITLE
Misc docs updates

### PR DESCRIPTION
**Pull request by jeromelaban**

### CODE REVIEW

File: doc/articles/features/using-the-uno-sdk.md [hunk 1]
```diff
@@ -35,36 +39,46 @@ In the csproj of an app, you will find the following property:
...
-- `Cupertino`, to enable Uno.Cupertino
-- `Mvvm`, to enable CommunityToolkit.Mvvm
-- `Prism`, to enable Prism Library
-- `Skia`, to enable SkiaSharp
-- `Svg`, to enable Svg support for iOS, Android, and mac Catalyst. This option is not needed for WebAssembly, Desktop, and WinAppSDK.
-- `Lottie`, to enable lottie files playback
+| feature | description |
+|---------|-------------|
+| `Foldable` | Adds a reference to Uno.WinUI.Foldable |
+| `MediaElement` | Adds a native references where needed to use MediaElement |
```
Comment (agneszitte) [inline doc/articles/features/using-the-uno-sdk.md:52]:
```suggestion
| `MediaElement` | Adds native references where needed to use MediaElement |
```

File: doc/articles/features/using-the-uno-sdk.md [hunk 2]
```diff
@@ -35,36 +39,46 @@ In the csproj of an app, you will find the following property:
...
+| `Storage` | Adds support for Storage using Uno.Extensions |
+| `Serialization` | Adds support for Uno.Extensions.Serialization |
+| `Toolkit` | Adds support for the Uno.Toolkit |
+| `Material` | Adds support for the Material Design Toolkit |
+| `Cupertino` | Adds support for the Cupertino Design Toolkit |
+| `Mvvm` | Adds support for the CommunityToolkit.Mvvm package |
+| `Dsp` | Adds support for the Uno.Dsp package |
+| `Prism` | Adds Prism support for Uno.WinUI |
+| `Skia` | Adds support for SkiaSharp |
+| `Svg` | Svg support for iOS, Android, and mac Catalyst. This option is not needed for WebAssembly, Desktop, and WinAppSDK. |
```
Comment (agneszitte) [inline doc/articles/features/using-the-uno-sdk.md:77]:
```suggestion
| `Svg` | Svg support for iOS, Android, and Mac Catalyst. This option is not needed for WebAssembly, Desktop, and WinAppSDK. |
```

File: doc/articles/upgrading-nuget-packages.md [hunk 3]
```diff
@@ -5,15 +5,11 @@ uid: Uno.Development.UpgradeUnoNuget
 
 Upgrading packages in your applications is done differently, depending on how your solution has created.
```
Comment (agneszitte) [inline doc/articles/upgrading-nuget-packages.md:6]:
```suggestion
Upgrading packages in your applications is done differently, depending on how your solution has been created.
```

File: doc/articles/upgrading-nuget-packages.md [hunk 4]
```diff
@@ -5,15 +5,11 @@ uid: Uno.Development.UpgradeUnoNuget
...
 
 Choose one of the sections below depending on your situation.
 
-## Projects without the Uno.Sdk
-
-To upgrade nuget packages without the Uno.Sdk, you can use the [Nuget Package Manager](https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio) coming from Visual Studio. Choose the latest stable versions of Uno Platform's NuGet packages.
-
 ## Projects using the Uno.Sdk
 
 Starting from Uno Platform 5.1 and using the new [Uno.Sdk](https://www.nuget.org/packages/uno.sdk), upgrading NuGet packages starting by `Uno.WinUI.` requires updating the `global.json` file at the root of your solution.
```
Comment (agneszitte) [inline doc/articles/upgrading-nuget-packages.md:15]:
```suggestion
Starting from Uno Platform 5.1 and using the new [Uno.Sdk](https://www.nuget.org/packages/uno.sdk), upgrading NuGet packages starting with `Uno.WinUI.` requires updating the `global.json` file at the root of your solution.
```

File: doc/articles/upgrading-nuget-packages.md [hunk 5]
```diff
@@ -55,3 +51,7 @@ To find the version to update to, pick the latest stable build from the [Uno.Win
 - [Uno.WinUI in nuget.org](https://www.nuget.org/packages/Uno.WinUI)
 
 Once the version has been chosen, change the `global.json` line with `"Uno.Sdk"` to use the newer version of Uno Platform. If you're running Visual Studio 2022, make sure to close/reopen the solution or restart the IDE.
+
+## Projects without the Uno.Sdk
+
+To upgrade nuget packages without the Uno.Sdk, you can use the [Nuget Package Manager](https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio) coming from Visual Studio. Choose the latest stable versions of Uno Platform's NuGet packages.
```
Comment (agneszitte) [inline doc/articles/upgrading-nuget-packages.md:57]:
```suggestion
To upgrade Nuget packages without the Uno.Sdk, you can use the [Nuget Package Manager](https://learn.microsoft.com/en-us/nuget/consume-packages/install-use-packages-visual-studio) coming from Visual Studio. Choose the latest stable versions of Uno Platform's NuGet packages.
```

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.2

**Pull request merged**